docs/genqlient.yaml

```diff
@@ -0,0 +1,114 @@
+# genqlient.yaml is genqlient's configuration file.  This genqlient.yaml is an
+# example; use `go run github.com/Khan/genqlient --init` to generate a simple
+# starting point.
+
+# The filename with the GraphQL schema (in SDL format), relative to
+# genqlient.yaml.
+schema: schema.graphql
+
+# Filenames or globs with the operations for which to generate code, relative
+# to genqlient.yaml.
+#
+# These may be .graphql files, containing the queries in SDL format, or
+# Go files, in which case any string-literal starting with (optional
+# whitespace and) the string "# @genqlient" will be extracted as a query.
+operations:
+- genqlient.graphql
+- "pkg/*.go"
+
+# The filename to which to write the generated code, relative to
+# genqlient.yaml.
+generated: generated/genqlient.go
+
+# The package name for the output code; defaults to the directory name of
+# the generated-code file.
+package: mygenerated
+
+# If set, a file at this path (relative to genqlient.yaml) will be generated
+# containing the exact operations that genqlient will send to the server.
+#
+# This is useful for systems which require queries to be explicitly
+# safelisted, especially for cases like queries involving fragments where
+# it may not exactly match the input queries, or for other static analysis.
```

Comment:
Is safelisting queries a standard GraphQL practice or a KA practice? Will others know why you might safelist queries?

Reply:
Ah good point.  I think its pretty common but I can link to e.g. https://www.apollographql.com/docs/studio/operation-registry/